python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -874,8 +916,23 @@ router bgp {{ router_bgp.as }}
 {%             elif peer_group.activate is arista.avd.defined(false) %}
       no neighbor {{ peer_group.name }} activate
 {%             endif %}
+{%             if peer_group.additional_paths.send is arista.avd.defined %}
+{%                 set af_ipv4_pg_add_path = 'neighbor ' ~ peer_group.name ~ ' additional-path send ' ~  peer_group.additional_paths.send %}
+{%                 if peer_group.additional_paths.send_ecmp_limit is arista.avd.defined and peer_group.additional_paths.send == 'ecmp' %}
+{%                     set af_ipv4_pg_add_path = af_ipv4_pg_add_path ~ ' limit ' ~ peer_group.additional_paths.send_ecmp_limit %}
+{%                 elif peer_group.additional_paths.send_limit is arista.avd.defined and peer_group.additional_paths.send == 'limit' %}
```

Comment:
there is an issue here, if user add send type as limit but not define the send_limit in that case we will have invalid command. So can you just check first we can handle this case then lets discuss first before to push.
cc: @ClausHolbechArista